rust/stackablectl/src/cli/mod.rs

```diff
@@ -213,7 +140,7 @@ pub enum Commands {
 
     /// Interact with deployed stacklets, which are bundles of resources and
     /// containers required to run the product.
-    #[command(aliases(["stl", "sl"]))]
+    #[command(aliases(["stl", "sl", "svc"]))]
```

Comment:
Wuhu :D I think I'm also happy to remove it

Reply:
I don't have a strong preference. We can keep it in for some time (to help users migrate) and drop it eventually in the future.

Reply:
If it's incompatible anyway then I don't think there's much value in keeping it.

Reply:
Removed in 07c786d